controllers/progressivesync_controller.go

```diff
@@ -394,19 +293,14 @@ func (r *ProgressiveSyncReconciler) removeAnnotationFromApps(ctx context.Context
 }
 
 // updateStageStatus updates the target stage given a stage name, message and phase
-func (r *ProgressiveSyncReconciler) updateStageStatus(ctx context.Context, name, message string, phase syncv1alpha1.StageStatusPhase, pr *syncv1alpha1.ProgressiveSync) error {
+func (r *ProgressiveSyncReconciler) updateStageStatus(ctx context.Context, name, message string, phase syncv1alpha1.StageStatusPhase, pr *syncv1alpha1.ProgressiveSync) {
```

Comment:
Why have we removed the `error` return value here?

Reply:
I think it's because `pr.SetStageStatus` doesn't return anything and we now make the update while calling `updateStatusWithRetry`.